presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hive-redirection-to-iceberg/iceberg.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=iceberg
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+iceberg.file-format=PARQUET
```

Comment:
irrelevant?

Reply:
The Iceberg Connector is needed to test the redirection.

Reply:
Surely. I was commenting on the last line of the config only (`iceberg.file-format=PARQUET`)